release.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
```

Comment:
Could not find a grunt task that did exactly what we want (in a simple way - either without branch or release) do I just scripted it for now - we might replace this later.